_hooks/pre-commit

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+
+# A git hook script that checks for trailing whitespace
```

Comment:
OK, structured the hook the same as bigsanity with one exception.  It appears as though having the hooks in a dir called ``hooks`` and then creating a symlink to it is causing jekyll to spit out errors when building locally as opened under the jekyll-watch gem here: https://github.com/jekyll/jekyll-watch/issues/36

I recommend we keep the hooks in ``_hooks`` instead of ``hooks`` for the time being until it is possible to exclude dirs that have symlinks from the jekyll-watcher.